src/blog/event-sourcing-vs-crud.md

```diff
@@ -0,0 +1,23 @@
+---
+title: From CRUD to Event Sourcing
+date: 2018-03-09
+author_github: jroper
+tags: event+sourcing crud
+summary: >
+    A side by side comparison of using Event Sourcing vs using CRUD
+---
+
+As the Lagom project has grown in usage, one of the biggest barriers to adoption that we've seen is Lagom's opinionated approach to using event sourcing for persistence. Many developers looking to write reactive microservices come from a CRUD background, often using relational databases to store data.
```

Comment:
Will everyone know the acronym "CRUD"? It might be one of those things that many people assume without having a name for it (like a [fish not knowing what water is](https://www.goodreads.com/quotes/187497-one-thing-about-which-fish-know-exactly-nothing-is-water)).

Maybe a good idea to link it to https://en.wikipedia.org/wiki/Create,_read,_update_and_delete or https://www.codecademy.com/articles/what-is-crud?